release.nix

```diff
@@ -6,7 +6,7 @@
 let
   source = sources.tezos;
   commonMeta = {
-    version = builtins.replaceStrings [ "refs/tags/v" ] [ "" ] meta.tezos_ref;
+    version = builtins.substring 1 100 (builtins.replaceStrings ["refs/tags/"] [""] meta.tezos_ref);
```

Comment:
I may be wrong, but IIRC we've had problems in the past when the tag/branch contained the `/` in its name, which means that it would be cut with your second suggestion.

So I went with the first one which might be more robust because of this